.github/workflows/template.module.yml

```diff
@@ -0,0 +1,114 @@
+name: '.Template: Module Pipeline'
+
+on:
+  workflow_call:
+    inputs:
+      workflowInput:
+        type: string
+        description: 'Input from the workflow caller in Json format'
+        required: true
+      moduleTestFilePaths:
+        type: string
+        description: 'List of relative path to the module test files in Json format'
+        required: true
+      modulePath:
+        type: string
+        description: 'Relative path to the module folder'
+        required: true
+
+env:
+  variablesPath: 'settings.yml'
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  ARM_SUBSCRIPTION_ID: '${{ secrets.ARM_SUBSCRIPTION_ID }}'
```

Comment:
I cannot recall - do we actually need them up here, or can we reference them also just directly?

I see we reference items below like:
```yaml
          subscriptionId: '${{ secrets.ARM_SUBSCRIPTION_ID }}'
          managementGroupId: '${{ secrets.ARM_MGMTGROUP_ID }}'
```
- not even using the `env:`

Reply:
I was looking into that as well, as it made no sense to me either. Finally, I noticed we use env var in GitHub actions, where the secrets collection is instead not accessible :D
So, I kept what is currently implemented in all workflows.